agent/tools.go

```diff
@@ -1,10 +0,0 @@
-//go:build tools
```

Comment:
Again, educate me: what was the point of this before?

Reply:
Ahh - this is explained here: https://github.com/determined-ai/determined/pull/3383